dotnet/dotnet-hello-world/src/Pages/Index.cshtml

```diff
@@ -0,0 +1,46 @@
+﻿@page
+@model IndexModel
+@{
+    ViewData["Title"] = "Congratulations | Cloud Run";
+}
+
+
+<div class="hero">
+    <div style="text-align:center;">
+        <picture>
+            <source srcset="img/KE-hello-world-dark.svg" alt="A group celebrating" media="(prefers-color-scheme: dark)">
+            <img src="img/KE-hello-world.svg" alt="A group celebrating" width="427" height="231">
```

Comment:
Check the address of this file. This image is under /wwwroot folder that you edited in this PR